autobahn/websocket/protocol.py

```diff
@@ -2146,11 +2146,11 @@ def sendCloseFrame(self, code=None, reasonUtf8=None, isReply=False):
         """
         if self.state == WebSocketProtocol.STATE_CLOSING:
             if self.debugCodePaths:
-                self.factory._log("ignoring sendCloseFrame since connection is closing")
```

Comment:
Ahh, hmm yeah, like if they had .debug()s as well you mean?

Certainly this whole PR is a step forward for Autobahn's logging, so bogging it down trying to *also* fix the .debugCodePaths, .debug_wamp etc things is silly; sorry for the noise ... but I'm still confused about what ``PassthroughLogger`` is really accomplishign.